k2/csrc/cuda_headers.cuh

```diff
@@ -0,0 +1,31 @@
+/**
+ * @brief
+ * Put cuda runtime headers here.
+ * With this file be included, code static analysis works.
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey
+ *                                                   Haowen Qiu)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#ifndef K2_CSRC_CUDA_HEADERS_H_
+#define K2_CSRC_CUDA_HEADERS_H_
+
+#ifdef __CUDA_ARCH__
+#define K2_CUDA_HOSTDEV __host__ __device__
```

Comment:
This macro is never used. Are there any negative effects if this file is removed?